.github/workflows/close-incomplete-issues.yml

```
@@ -0,0 +1,15 @@
+name: Close incomplete issue templates
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  close-issues-if-invalid:
+    steps:
+      - uses: ddbeck/invalid-issue-closer@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          labels: 'invalid :no_entry_sign:'
+          comment: 'This issue was automatically closed because it appears that the issue template has not been completed. If this has been closed in error, edit the title and body, and post a follow-up comment.'
```

Just a thought to make the comment a bit more verbose given the people who file empty issues need extra hand holding?

```suggestion
          comment: 'This issue was automatically closed because it appears to be empty. The issue is not different from the issue template and no actionable information has been provided. If this has been closed in error, edit the title and body, and post a follow-up comment.'
```

---

Good idea. Applied!